src/transformers/pipelines/audio_utils.py

```diff
@@ -127,6 +141,7 @@ def ffmpeg_microphone_live(
     stride_length_s: Optional[Union[Tuple[float, float], float]] = None,
     format_for_conversion: str = "f32le",
     ffmpeg_input_device: Optional[str] = None,
+    ffmpeg_additional_args: Optional[list] = [],
```

Comment:
Same comment here - no mutable object defaults 

```suggestion
    ffmpeg_additional_args: Optional[List[str]] = None,
```